modules/phase_field/src/kernels/ACBulk.C

```diff
@@ -10,44 +10,59 @@ template<>
 InputParameters validParams<ACBulk>()
 {
   InputParameters params = validParams<KernelValue>();
-  params.addClassDescription("Allen-Cahn Kernel");
+  params.addClassDescription("Allen-Cahn bulk energy kernel");
   params.addParam<MaterialPropertyName>("mob_name", "L", "The mobility used with the kernel");
+  params.addCoupledVar("args", "Vector of additional arguments to mobility");
```

Comment:
The doc string should reflect that these are the arguments to **both** mobility and free energy.